ci-operator/step-registry/dpu-operator/workflow/tests/dpu-operator-tests-commands.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+ssh root@wsfd-advnetlab144.anl.eng.bos2.dc.redhat.com dnf install git go
+ssh root@wsfd-advnetlab144.anl.eng.bos2.dc.redhat.com git clone https://github.com/openshift/dpu-operator.git
+ssh root@wsfd-advnetlab144.anl.eng.bos2.dc.redhat.com cd dpu-operator
+ssh root@wsfd-advnetlab144.anl.eng.bos2.dc.redhat.com make test
```

Comment:
Normally tests should be built in an image and run from there, this won't work on older OCP releases, pull requests, etc.

Reply:
I understand that tests should be built from an image. This is simply a test to ensure that builder05 can access our internal redhat labs and perform basic functions. Adding complexity would come later.